polaris-react/src/components/Frame/components/Toast/Toast.scss

```diff
@@ -28,7 +28,7 @@ $Backdrop-opacity: 0.88;
 
 .Action {
   margin-left: var(--p-space-400);
-  color: var(--p-color-text-on-color);
+  color: var(--p-color-text-inverse);
```

Comment:
other text was also updated to `inverse` in this pr https://github.com/Shopify/polaris/issues/10386